### DISCUSSION TITLE
烂尾楼问题能解决吗？我们到底该怎么办？这是要不了了之了吗？

### DISCUSSION THREAD

**Discussion by blisee** [WeNeedHome/SummaryOfLoanSuspension#934] [General]
国家没有新动作了，一切变的悄无声息了，感到绝望。。

**Comment by fanren112233** [comment]
你还没停贷呢 你不会不敢把

**Reply by blisee** [reply]
停贷就解决了吗

**Reply by xiejun946** [reply]
看吧，半年3个月之内交不了就停吧，没必要当冤大头。征信也无所谓了。请律师打官司要花一万多

**Comment by C-Joey** [comment]
你们楼盘不停贷 不给政府压力 它为什么要给你解决问题 
就比如上海封城的时候冲击小区大门 政府立马就发食物过来 
你们不停贷说明还能承受 还不是不稳定因素 政府慌什么

**Reply by blisee** [reply]
政府确实够稳重的

**Comment by pengyidong** [comment]
按闹分配

**Comment by ppinkiris** [comment]
闹啊 我们没被烂尾的能做的贡献就是打死不买期房，你们能断贷的统一断贷，就算大家一起玩完，也是我们这些没买房的纳税人给你们兜底，怕什么

**Comment by yooq** [comment]
那你就继续还烂尾贷款，等他良心发现来帮你解决呗。
